app/assets/javascripts/modules/api_users.js.coffee

```diff
@@ -23,10 +23,7 @@ class app.ApiUsers
     appendRow(row)
 
   apiUsersData = (data) ->
-    apiUsers = data['data']
-    if(!apiUsers)
-      apiUsers = [data['data']]
-    apiUsers
+    data['data']
```

Comment:
You tested this thoroughly right? This could break a couple of things i imagine

Reply:
Yes, I think it is thoroughly tested. I assume that every test case is covered in this [test](https://github.com/puzzle/cryptopus/blob/feature-54339-fix-api-user-bug/spec/controllers/api/api_users/lock_controller_spec.rb) as well in this [one](https://github.com/puzzle/cryptopus/blob/feature-54339-fix-api-user-bug/spec/controllers/api/api_users_controller_spec.rb). The Problem here was that the JSON structure didn't match the parameter of the [apiUsersContent](https://github.com/puzzle/cryptopus/blob/a0a66d32910a6249a610945e78656a026185c121/app/assets/javascripts/modules/api_users.js.coffee#L17) Method. I fixed this by adding Brackets to the loaded data, when a new ApiUser gets created.